commit 14f53ca0a3b47f5584172d2ac46630e3517e05f1 (refs/changes/58/2158/3)
Author: David Pacheco <dap@joyent.com>
Date:   2017-06-27T15:49:51-07:00 (2 years, 3 months ago)
    
    MANTA-3332 manta README out of date after MANTA-3288
    Reviewed by: Trent Mick <trentm@gmail.com>
    Reviewed by: Kody A Kantor <kody.kantor@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>
    Approved by: Kody A Kantor <kody.kantor@gmail.com>

diff --git a/README.md b/README.md
index a712a8f..c187829 100644
--- a/README.md
+++ b/README.md
@@ -42,8 +42,7 @@ the user documentation for details.
 * To see a detailed, real example of using Manta, check out [Kartlytics: Applying Big Data Analytics to Mario Kart](http://www.joyent.com/blog/introducing-kartlytics-mario-kart-64-analytics).
 
 * To learn about installing and operating your own Manta deployment, see the
-[Manta Operator's
-Guide](https://github.com/joyent/manta/blob/master/docs/operator-guide/index.md).
+[Manta Operator's Guide](https://joyent.github.io/manta/).
 
 * To understand Manta's architecture, see
 [Bringing Arbitrary Compute to Authoritative
